README.md

```diff
@@ -1 +1,18 @@
-# dcf-dataservice
\ No newline at end of file
+# dcf-dataservice
+Copy data buckets from GDC to AWS and Google buckets. The jobs need to be run inside a setup kubeneste cluster.
```

Comment:
```suggestion
 Copy data buckets from GDC to AWS and Google buckets. The jobs need to be run inside a setup kubernetes cluster.
```

Reply:
fixed in d47a36c889171c98bbe27a87e89fb6d55d356f0f